aikau/src/main/resources/alfresco/util/hashUtils.js

```diff
@@ -113,6 +113,7 @@ define(["dojo/_base/array",
        * a suppresssDecoding argument of true is provided.
        *
        * @instance
+       * @function
```

Comment:
Have you checked what it does to the JSDoc output. Things might have changed since I first got the JSDoc outputting nicely, but we needed to use certain tags to get the output looking correct - I just want to make sure this doesn't change the output unexpectedly.